src/backend/src/transformers/reimbursement-requests.transformer.ts

```diff
@@ -0,0 +1,49 @@
+import { Prisma } from '@prisma/client';
+import { ReimbursementRequest } from 'shared';
+import reimbursementRequestQueryArgs from '../prisma-query-args/reimbursement-requests.query-args';
+import { wbsNumOf } from '../utils/utils';
+import userTransformer from './user.transformer';
+
+const reimbursementRequestTransformer = (
+  reimbursementRequest: Prisma.Reimbursement_RequestGetPayload<typeof reimbursementRequestQueryArgs>
+): ReimbursementRequest => {
+  const { vendor, expenseType } = reimbursementRequest;
+  return {
+    reimbursementRequestId: reimbursementRequest.reimbursementRequestId,
+    saboId: reimbursementRequest.saboId ?? undefined,
+    dateCreated: reimbursementRequest.dateCreated,
+    dateDeleted: reimbursementRequest.dateDeleted ?? undefined,
+    dateOfExpense: reimbursementRequest.dateOfExpense,
+    reimbursementsStatuses: reimbursementRequest.reimbursementsStatuses.map((reimbursementStatus) => ({
+      reimbursementStatusId: reimbursementStatus.reimbursementStatusId,
+      type: reimbursementStatus.type,
```

Comment:
you may need to make a function to map a prisma status to a shared status

Reply:
Might not be needed anymore